packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1162,7 +1194,8 @@ export default (Commands, Cypress, cy, state, config) => {
         // so that we nuke the previous state. subsequent
         // visits will not navigate to about:blank so that
         // our history entries are intact
-        if (!hasVisitedAboutBlank) {
+        // TODO: is this okay to skip for multi-domain?
```

Comment:
I feel like that is a safe assumption. How do users visit `about:blank` within the context of multi-domain?

Reply:
I don't want to start two different conversations so I'm going to resolve this one and we can discuss here https://github.com/cypress-io/cypress/pull/20467#discussion_r819028009